### PR TITLE
GSYE-380: Add WindTurbineValidator and wind_turbine to scenario valid…

### PR DESCRIPTION
**Pull request by hannesdiedrich**
…ator

### CODE REVIEW

File: gsy_framework/schemas.py [hunk 1]
```diff
@@ -81,7 +82,33 @@ class ScenarioSchemas:
...
+                    "use_market_maker_rate": {"type": "boolean"},
+                    "powerProfileUUID": {"type": "string"}
+                }
+            },
+            "wind_turbine": {
+                "type": "object",
+                "properties": {
+                    "name": {"type": "string"},
+                    "type": {"enum": ["WindTurbine"]},
+                    "number_of_clones": {"type": "number"},
```
Comment (spyrostz) [inline gsy_framework/schemas.py:94]: I think that this can be removed

**Review by spyrostz — COMMENTED** [review]
One comment, LGTM once resolved. Probably my comment here:
https://github.com/gridsingularity/gsy-web/pull/1787#discussion_r970505636 
will also affect this PR though

File: gsy_framework/schemas.py [hunk 2]
```diff
@@ -79,7 +80,32 @@ class ScenarioSchemas:
...
+                                                                  {"type": "null"}]},
+                    "power_profile": {
+                        "anyOf": [
+                            {"type": "object"},
+                            {"type": "number"},
+                            {"type": "null"},
+                            {"type": "array"},
+                            {"type": "string"}]},
+                    "use_market_maker_rate": {"type": "boolean"},
+                    "powerProfileUUID": {"type": "string"}
```
Comment (spyrostz) [inline gsy_framework/schemas.py:108]: Should the capacity_kW not be included here too, in order to pass through the schema validator ?

**Review by spyrostz — APPROVED** [review]
One question / comment, LGTM once resolved

File: gsy_framework/schemas.py (hunk 2 above)
Comment (hannesdiedrich) [reply gsy_framework/schemas.py:108]: It should indeed. Shows that I did not test it end to end again 🙈 . Let me add it

**Review by spyrostz — APPROVED** [review]
LGTM

**Review by ERafaelMartinez — APPROVED** [review]
LGTM

**Pull request merged**